test failure looks related.

```
======================================================================
FAIL: corehq.messaging.scheduling.tests.test_content:TestContent.test_sms_language_fallback
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/messaging/scheduling/tests/test_content.py", line 190, in test_sms_language_fallback
    ''
AssertionError: 'non-translated message' != ''
- non-translated message
+ 
```

--- a/corehq/messaging/scheduling/models/abstract.py
+++ b/corehq/messaging/scheduling/models/abstract.py
@@ -444,8 +444,7 @@ class Content(models.Model):
         :param preferred_language_code: the language code of the user's preferred language
         """
 
-        # If: message_dict contains no translations i.e. only contains * ;
-        # Then: treat it as an alert and send the message.
+        # return untranslated content, if no translations set
         if {'*'} == message_dict.keys():
             return Content.get_cleaned_message(message_dict, '*')
 

--- a/corehq/messaging/scheduling/tests/test_content.py
+++ b/corehq/messaging/scheduling/tests/test_content.py
@@ -150,6 +150,7 @@ class TestContent(TestCase):
             'en': 'english message',            # project default
             'hin': 'hindi message',             # schedule default
             'es': 'spanish message',            # user's preferred language
+            'kan': 'kannada message',           # arbitrary language to test untranslated case
         }
         user_lang = self.user.get_language_code()
         content = Content()
@@ -195,6 +196,7 @@ class TestContent(TestCase):
             content.get_translation_from_message_dict(self.domain_obj, message_dict, user_lang),
             message_dict['*']
         )
+        message_dict.pop('kan')
 
         # Default: same as LANGUAGE_FALLBACK_UNTRANSLATED
         self.domain_obj.sms_language_fallback = None

